services/net/capture/appsettings.json

```diff
@@ -14,7 +14,7 @@
   "Service": {
     "MaxFailLimit": 5,
     "ApiUrl": "http://api:8080",
-    "TimeZone": "UTC",
+    "TimeZone": "PST",
```

Comment:
The services will default to PST so that the comparison and formatting of dates is correct.  Not sure if this will impact other logic in the service yet.